README.md

```diff
@@ -0,0 +1,90 @@
+# Navitia PHP client
+
+PHP client for Navitia API (http://navitia.io).
+
+
+## Installation
+
+Requires PHP 5.4 or +
+
+Add in your composer:
+
+``` json
+{
+    "require": {
+        "canaltp/navitiaphp": "~1.0"
+    }
+}
+```
+
+Then `composer update`.
+
+
+## Usage
+
+Instanciation and call your first Api endpoint (i.e `coverage`):
+
+``` php
+require_once 'vendor/autoload.php';
+
+use CanalTP\NavitiaPhp\Navitia;
+
+$client = Navitia::createFromBaseUrlAndToken('http://api.navitia.io/v1/', '3b036afe-0110-4202-b9ed-99718476c2e0');
```

Comment:
Maybe 
`$navitia = Navitia::createFromBaseUrlAndToken('http://api.navitia.io/v1/', '3b036afe-0110-4202-b9ed-99718476c2e0');` would be less confusing to a new user ; because Navitia::createFromBaseUrlAndToken() doesn't return a "http" client but a navitia instance.